devenv.project.properties

```diff
@@ -77,8 +63,9 @@ IOM_APP_IMAGE=
 # be ignored.
 # examples:
 # local image:                      iom:4.1.2
+# locally built project image:      prj-iom:4.1.2-1.2.0.0-SNAPSHOT
 # IOM release from ISH registry:    docker.intershop.de/intershophub/iom:4.1.2
-IOM_IMAGE=docker.io/library/iom-blueprint-project-iom:1.4.0
+IOM_IMAGE=docker.io/library/iom-blueprint-project-iom:1.5.0-SNAPSHOT
```

Comment:
Partners do not have access to SNAPSHOT images. Therefore the configuration has to use release versions of IOM only.